returnn/datasets/hdf.py

```diff
@@ -273,6 +274,27 @@ def get_data(self, seq_idx, key):
 
     # Otherwise, directly read it from file now.
     real_seq_idx = self._seq_index[seq_idx]
+    return self._get_data_by_real_seq_idx(real_seq_idx, key)
+
+  def get_data_by_seq_tag(self, seq_tag, key):
+    """
+    :param str seq_tag:
+    :param str key:
+    :rtype: numpy.ndarray
+    """
+    if self.cache_byte_size_total_limit > 0:  # Use the cache?
+      assert False  # what should we do here?
```

Comment:
Throw a better exception instead, like `raise Exception("%s: get_data_by_seq_tag not supported with cache" % self)` or so.